tests/e2e/cucumber/step_definitions/app.files.ts

```diff
@@ -394,3 +395,36 @@ When(
     }
   }
 )
+
+When(
+  'the unauthorized user access the last public link created',
+  async function (this: World): Promise<void> {
+    const publicUser = new PublicLinkPage()
+    await publicUser.navigateToPublicLink()
+  }
+)
+
+When(
+  'the unauthorized user authenticates with correct password {string} to access public link',
+  async function (this: World, password: string): Promise<void> {
+    const publicUser = new PublicLinkPage()
+    await publicUser.authenticatePassword(password)
+  }
+)
+
+Then(
+  'the following file should not be visible on the public link',
```

Comment:
could you make the method more universal? 
`the following (file|files) should (be|not be) visible on the public link`